src/analyses/region.ml

```diff
@@ -70,16 +70,16 @@ struct
       (ps, es)
 
   (* queries *)
-  let query ctx (q:Queries.t) : Queries.Result.t =
+  let query ctx (type a) (q: a Queries.t): a Queries.result =
     let regpart = get_regpart ctx in
     match q with
     | Queries.Regions e ->
-      if is_bullet e regpart ctx.local then `Bot else
+      if is_bullet e regpart ctx.local then Queries.Result.bot q (* TODO: remove bot *) else
```

Comment:
why bot? This seems strange, as well will meet these and one bot makes everything bot here?